### ISSUE TITLE
We need to determine the naming conventions that we want to use for courses and assignments

### ISSUE THREAD

**Issue by lwasser** [earthlab/autograding-notebooks#6]
determine naming conventions for assignments

i suggest for courses we use:
`earth-analytics-`
`earth-analytics`
`earth-analytics-applications`
`earth-analytics-bootcamp`

for assignments i guess we won't have the students modify the notebook and add their name but we might need the course name in there as all of the repos are together so something like...

Jenny's course
`ea-bootcamp-01-assignment-name`
Leah's course:
`ea-01-spatial-vector-data`
`ea-07-fire-remote-sensing-images`

where the number relates tot he week number to allow for ordering and the name begins with an acronym for the course like ea-bootcamp or ea-boot . 

Thoughts @jlpalomino @kcranston

**Comment by kcranston** [comment]
That sounds good to me. GitHub classroom adds the username to student repos, so they will be  (for example): 

`ea-bootcamp-01-spatial-vector-data-kcranston`

The scripts currently assume that the assignment name in nbgrader matches the assignment name in github classroom, and it seems simplest to keep it that way. The name of the template repo has no constraints, but for consistency, I suggest `coursename-##-assignment-template`, e.g.:

`ea-bootcamp-01-spatial-vector-data-template`

**Comment by jlpalomino** [comment]
Agreed! Since I only had four homeworks in the short version of the course, I didn't name the assignments but just used a number (e.g. hw-1). However, this year, I am able to give each week's assignment a proper name that would fit this naming convention. 

The final project is the only repo that wouldn't need a number, unless we gave it the number for the last week of class, so either: ```ea-bootcamp-final-project``` or ```ea-bootcamp-15-final-project```.

**Comment by lwasser** [comment]
wonderful!! ok so we can add this to the hackmd doc?? slowly build out our "rules"

i like using hackmd when i'm not sure where all of the docs should go. i could see this in a readme for the template repo or something too.. but again all in one place to begin.

**Comment by kcranston** [comment]
Added to hackmd: https://hackmd.io/_4GDiG9wSwe6lBq18bDM7w?view#Naming-conventions

Please review, and then we can close this. I will need to make a few changes to the classroom-nbgrader integration scripts once we finalize.

**Comment by kcranston** [comment]
One thought is to add `nbgrader` to the name of the nbgrader repo so that it is easy to pick these out from among the many repos in the classroom organization.

**Comment by lwasser** [comment]
+1 @kcranston i did add that by default when i created it locally once i realized it was the course repo for grading.

**Comment by kcranston** [comment]
Are we all happy with the naming conventions at https://hackmd.io/_4GDiG9wSwe6lBq18bDM7w?view#Naming-conventions (I added the nbgrader convention).

**Comment by lwasser** [comment]
i think yes @kcranston it looks good to me. do you want to close this? i'm fine with that if you do!
